### ISSUE TITLE
add events "addItemEvent" and "removeItemEvent"

### ISSUE THREAD

**Issue by scalpatt** [jbaysolutions/vue-grid-layout#399]
Hi,

Can you add this two events ? or emit updateLayout if an item is added or remmoved ? 

Thnaks.

**Comment by gmsa** [comment]
What would be the use case for this? If you're adding or removing an item programmatically do you really need the events?

**Comment by gmsa** [comment]
@mkgn you can use the Vue instance lifecycle hooks for that: https://vuejs.org/v2/guide/instance.html#Instance-Lifecycle-Hooks

**Comment by mkgn** [comment]
Yes you are correct. Having Add/Remove event won't do much. Thank you! We can close this.
